nixos/modules/services/backup/borgbackup.nix

```diff
@@ -555,6 +553,19 @@ in {
             example = "--save-space";
           };
 
+          extraPackages = mkOption {
```

Comment:
What do you think about calling this `path`? I'm not sure what's better.

Reply:
I think `path` is better as this option appends packages to the `path` option of the systemd service.